data.sql

```diff
@@ -122,15 +132,69 @@ INSERT INTO animals (
   )
 VALUES(11, 'Ditto', '2022-05-14', 4, TRUE, 22, null);
 
-INSERT INTO owners (full_name, age) VALUES ('Sam Smith', 34), ('Jennifer Orwell', 19), ('Bob', 45), ('Melod
-y Pond', 77), ('Dean Winchester', 14), ('Jodie Whittaker', 38);
-INSERT INTO species (name) VALUES ('Pokemon'), ('Digimon');
+INSERT INTO owners (full_name, age)
+VALUES ('Sam Smith', 34),
+  ('Jennifer Orwell', 19),
+  ('Bob', 45),
+  ('Melod
+y Pond', 77),
+  ('Dean Winchester', 14),
+  ('Jodie Whittaker', 38);
+INSERT INTO species (name)
+VALUES ('Pokemon'),
+  ('Digimon');
+
+UPDATE animals
+SET species_id = 2
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species_id = 1
+WHERE name NOT LIKE '%mon';
+
+UPDATE animals
+SET owner_id = 1
+WHERE name = 'Agumon';
+
+UPDATE animals
+SET owner_id = 2
+WHERE name = 'Gabumon'
+  OR name = 'Pikachu';
+
+UPDATE animals
+SET owner_id = 3
+WHERE name = 'Devimon'
+  OR name = 'Plantmon';
+
```

Comment:
Please Insert the following data for vets:
`    
    Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
    Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
    Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
    Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.
`